v2/pkg/cli/executor.go

```diff
@@ -44,22 +44,59 @@ import (
 var (
 	mirrorlongDesc = templates.LongDesc(
 		` 
-		Create and publish user-configured mirrors with a declarative configuration input.
-		used for authenticating to the registries. 
+		Mirror OCP Release, operator catalog and additional images using a declarative configuration file as an input. 
 
-		The podman location for credentials is also supported as a secondary location.
+		The command line uses a declarative configuration file as the input to discover from where to get the container images and copy them to the destination specified in the command line. 
 
-		1. Destination prefix is docker:// - The current working directory will be used.
-		2. Destination prefix is oci:// - The destination directory specified will be used.
+		There are three workflows available in oc-mirror currently:
+			
+			- mirrorToDisk - pulls the container images from the source specified in the image set configuration and pack them into a tar archive on disk (local directory).
```

Comment:
```suggestion
			- mirrorToDisk - pulls the container images from the source specified in the image set configuration and packs them into a tar archive on disk (local directory).
```

Reply:
Done :white_check_mark:

Thanks for the suggestion.